doc/sphinx/addendum/type-classes.rst

```diff
@@ -381,37 +371,71 @@ Summary of the commands
 Besides the :cmd:`Class` and :cmd:`Instance` vernacular commands, there are a
 few other commands related to typeclasses.
 
-.. cmd:: Existing Instance {+ @ident} {? | @natural}
+.. cmd:: Existing Instance @qualid {? @hint_info }
+         Existing Instances {+ @qualid } {? %| @natural }
 
-   This command adds an arbitrary list of constants whose type ends with
+   .. insertprodn hint_info hint_info
+
+   .. prodn::
+      hint_info ::= %| {? @natural } {? @one_term }
+
+   This command adds a constant whose type ends with
    an applied typeclass to the instance database with an optional
    priority :token:`natural`.  It can be used for redeclaring instances at the end of
    sections, or declaring structure projections as instances. This is
    equivalent to ``Hint Resolve ident : typeclass_instances``, except it
    registers instances for :cmd:`Print Instances`.
 
-.. tacn:: typeclasses eauto
-   :name: typeclasses eauto
+.. cmd:: Print Instances @reference
+
+   Shows the list of currently registered relations, Equivalences, PreOrders, PERs,
+   and Morphisms (implemented as ``Proper`` instances) of the type specified by
+   :token:`reference`.
+   :token:`reference`` can be ``Reflexive``, ``Symmetric`` or ``Transitive``.
```

Comment:
There is a build error because of the additional backtick:

```suggestion
   :token:`reference` can be ``Reflexive``, ``Symmetric`` or ``Transitive``.
```